lib/db_helper/sql_importer.rb

```diff
@@ -2,158 +2,160 @@
+          :socket   => get_input(:prompt => "Socket: ",   :allow_blank => true)
+        }
+      end
 
-    src_sql = MysqlInterface.new(src_db_config)
-    dst_sql = MysqlInterface.new(destination_db_config)
-    dst_sql.drop_and_create if included_tables.empty?
+      src_sql = MysqlInterface.new(src_db_config)
+      dst_sql = MysqlInterface.new(destination_db_config)
+      dst_sql.drop_and_create if included_tables.empty?
 
-    puts "1. Backing up database"
+      puts "1. Backing up database"
 
-    mysql_cmd = "mysqldump #{src_sql.options} #{src_sql.database}"
+      mysql_cmd = "mysqldump #{src_sql.options} #{src_sql.database}"
 
-    dump_data_cmd = "#{mysql_cmd} %s %s > %s" % [
-      included_tables.map{|t| " #{t}"}.join,
-      excluded_tables.map{|t| " --ignore-table=#{src_sql.database}.#{t}"}.join,
-      "/tmp/#{data_file_name}"
-    ]
+      dump_data_cmd = "#{mysql_cmd} %s %s > %s" % [
+        included_tables.map{|t| " #{t}"}.join,
+        excluded_tables.map{|t| " --ignore-table=#{src_sql.database}.#{t}"}.join,
+        "/tmp/#{data_file_name}"
+      ]
 
-    compress_data_cmd = "gzip '/tmp/#{data_file_name}'"
+      compress_data_cmd = "gzip '/tmp/#{data_file_name}'"
 
-    dump_structure_cmd = "#{mysql_cmd} --no-data --tables #{excluded_tables.join(" ")} > /tmp/#{structure_file_name}" if included_tables.empty? && !excluded_tables.empty?
+      dump_structure_cmd = "#{mysql_cmd} --no-data --tables #{excluded_tables.join(" ")} > /tmp/#{structure_file_name}" if included_tables.empty? && !excluded_tables.empty?
 
-    [dump_data_cmd, dump_structure_cmd, compress_data_cmd].compact.each do |dump_cmd|
-      command "ssh -p #{ssh_port} #{ssh_user}@#{ssh_host} \"#{dump_cmd}\""
-    end
+      [dump_data_cmd, dump_structure_cmd, compress_data_cmd].compact.each do |dump_cmd|
+        command "ssh -p #{ssh_port} #{ssh_user}@#{ssh_host} \"#{dump_cmd}\""
+      end
 
 
-    puts "2. Coping & Decompressing SQL Files..."
-    [data_file_name + ".gz", structure_file_name].compact.each_with_index do |f, index|
-      command "scp -P #{ssh_port} #{ssh_user}@#{ssh_host}:/tmp/#{f} #{f}"
-      command("ssh -p #{ssh_port} #{ssh_user}@#{ssh_host} \"rm /tmp/#{f}\"")
-      command "gunzip #{f}" if f.end_with?('.gz')
-    end
+      puts "2. Coping & Decompressing SQL Files..."
+      [data_file_name + ".gz", structure_file_name].compact.each_with_index do |f, index|
+        command "scp -P #{ssh_port} #{ssh_user}@#{ssh_host}:/tmp/#{f} #{f}"
+        command("ssh -p #{ssh_port} #{ssh_user}@#{ssh_host} \"rm /tmp/#{f}\"")
+        command "gunzip #{f}" if f.end_with?('.gz')
+      end
 
-    puts "3. Importing SQL files..."
-    [data_file_name + ".gz", structure_file_name].compact.each_with_index do |f, index|
-      dst_sql.import(f.gsub('.gz', ''))
+      puts "3. Importing SQL files..."
+      [data_file_name + ".gz", structure_file_name].compact.each_with_index do |f, index|
+        dst_sql.import(f.gsub('.gz', ''))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.